Tests.ReactView/DefaultStylesheetLoadTests.cs

```diff
@@ -7,16 +7,26 @@ namespace Tests.ReactView {
 
     public class DefaultStyleSheetLoadTests : ReactViewTestBase {
 
+        private static ReactViewWithStyleSheet currentView;
```

Comment:
instead of these static things expose stuff through ReactViewWithStyleSheet  as instance methods

Reply:
Ok, but it will imply some casts.